docs/typescript/how-to-develop-a-worker-program-in-typescript.md

```diff
@@ -18,29 +19,29 @@ Below is an example of starting a Worker that polls the Task Queue named `tutori
 `taskQueue` is the only required option, but you will also use `workflowsPath` and `activities` to register Workflows and Activities with the Worker.
 See below for more Worker options.
 
-### Workflow and Activity registration
+#### Workflow and Activity registration
 
 Workers bundle Workflow code and `node_modules` using Webpack v5 and execute them inside V8 isolates.
 Activities are directly required and run by Workers in the Node.js environment.
 
-Workers are very flexible - you can host any or all of your Workflows and Activities on a Worker, and you can host multiple Workers in a single machine.
+Workers are very flexible – you can host any or all of your Workflows and Activities on a Worker, and you can host multiple Workers in a single machine.
 
 There are three main things the Worker needs:
 
 - `taskQueue`: the Task Queue to poll. This is the only required argument.
-- `activities`: Optional. Imported and supplied directly to the Worker. Not file path name.
+- `activities`: Optional. Imported and supplied directly to the Worker. Not the path.
 - Workflow bundle:
-  - Either specify a `workflowsPath` to your `workflows.ts` file to pass to Webpack, e.g. `require.resolve('./workflows')`. Workflows will be bundled with their dependencies, which you can finetune with `nodeModulesPaths`.
-  - Or pass a prebuilt bundle to `workflowBundle` instead if you prefer to handle the bundling yourself.
+- Either specify a `workflowsPath` to your `workflows.ts` file to pass to Webpack, e.g., `require.resolve('./workflows')`. Workflows will be bundled with their dependencies, which you can fine-tune with `nodeModulesPaths`.
+- Or pass a prebuilt bundle to `workflowBundle` instead if you prefer to handle the bundling yourself.
 
-### Additional Worker Options
+#### Additional Worker Options
```

Comment:
I think we will show case worker options in the context of other features or situations - like worker tuning for example 

So keep this info handy and on-deck but lets prob scrape it from mvp Workers section.